nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Col2Im.java

```diff
@@ -61,6 +61,13 @@ public Col2Im(SameDiff sd, SDVariable input, Conv2DConfig config){
 
     public Col2Im() {}
 
+    public Col2Im(INDArray[] in, Conv2DConfig conv2DConfig) {
```

Comment:
As per the dl4j-dev-tools PR: this should be `Col2Im(@NonNull INDArray input`, ... not INDArray[]